src/Emulator/Main/Peripherals/Input/KeyScanCode.cs

```diff
@@ -83,7 +83,7 @@ public enum KeyScanCode
         OemSemicolon = 0x33,
         OemQuotes = 0x34,
         CapsLock = 0x39,
-        OemPeriod= 0x36,
+        OemPeriod = 0x36,
```

Comment:
Could you move this formatting change to a separate commit?

Reply:
Split into 260ea611b8c6d661459e11f09032e6a1a7275ecd